packages/mui-material/src/InputLabel/InputLabel.js

```diff
@@ -109,8 +109,8 @@ const InputLabelRoot = styled(FormLabel, {
     ...(ownerState.shrink && {
       userSelect: 'none',
       pointerEvents: 'auto',
-      maxWidth: 'calc(133% - 24px)',
-      transform: 'translate(14px, -9px) scale(0.75)',
+      maxWidth: 'calc(150% - 24px)',
+      transform: 'translate(12px, -9px) scale(0.70)',
```

Comment:
Why this change? Let's not change this.

Reply:
@ZeeshanTamboli sorry this changes were related to another pull request